content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -84,7 +84,7 @@ owner object:
 
 After the owner object enters the *deletion in progress* state, the controller
```

Comment:
I think the "controller" here also needs to be made explicit as "garbage collection".

Reply:
> I think the "controller" here also needs to be made explicit as "garbage collection".

Thank you, I updated the PR